### DISCUSSION TITLE
MainLayout CSS isolation

### DISCUSSION THREAD

**Discussion by RoySalisbury** [microsoft/fluentui-blazor#991] [Q&A]
I'm trying to use CSS isolation for different site layouts.  Each layout may have a different FluentLayout with header/footer.

I tried moving the css for the footer out of app.css and into a MainLayout.razor.css file for isolation, but not all aspects of it get applied.  The main "footer" class never gets picked up.  The child content seems to, but not the main tag.

Any ideas?

**Comment by vnbaaij** [comment]
You probably need to add `::deep` selector to get the styles applied. 

Remember, `::deep` only works if the selectors you are targetting are inside an HTML element. You might need to add an enclosing `<div>` here or there,

**Reply by Lindsay-Mathieson** [reply]
> Remember, `::deep` only works if the selectors you are targetting are inside an HTML element. You might need to add an enclosing `<div>` here or there,

Thank you so much! That was what I was missing in my CSS Isolation style, have been tearing my hair out over it for days
